binaries/README.md

```diff
@@ -14,6 +14,31 @@ recorded with `prio` 0.4. That test is implemented in `tests/backward_compatibil
 vector was generated using `generate-test-vector`. See that tool's usage text for more information,
 and module `prio::test_vector` for utilities for working with test vectors.
 
+## `crypt`: encrypt and decrypt Prio v2 inputs
+
+`crypt` transforms inputs (a vector of `FieldPriov2`) into encrypted input shares as well as
+transforming encrypted input shares back into inputs. To decrypt input shares:
+
+    cargo run --bin crypt -- \
+        -d 10 \
+        --server-1-private-key "BIl6j+J6dYttxALdjISDv6ZI4/VWVEhUzaS05LgrsfswmbLOgNt9HUC2E0w+9RqZx3XMkdEHBHfNuCSMpOwofVSq3TfyKwn0NrftKisKKVSaTOt5seJ67P5QL4hxgPWvxw==" \
```

Comment:
IMO, don't put sensitive data like private keys into command-line arguments -- for manual usage, this is begging for someone to forget to `unset HISTFILE` & end up with a private key written to their `.bash_history`.

Perhaps `--server-{1,2}-private-key-file` which specifies a file with the private key?